django_extensions/management/commands/syncdata.py

```diff
@@ -157,20 +161,23 @@ def syncdata(self, fixture_labels, options):
                                 print("Installing %s fixture '%s' from %s." % (format, fixture_name, humanize(fixture_dir)))
                             try:
                                 objects_to_keep = {}
-                                objects = serializers.deserialize(format, fixture)
+                                objects = list(serializers.deserialize(format, fixture))
```

Comment:
Won't it be slightly more efficient to use the `objects_to_keep` in the subsequent runs instead of forcing the materialization of `objects` here ?